src/CommunityToolkit.Maui.Core/CommunityToolkit.Maui.Core.csproj

```diff
@@ -44,6 +44,12 @@
     <Configurations>Debug;Release</Configurations>
   </PropertyGroup>
 
+  <UsingTask TaskName="VerifyMauiWorkloadVersionTask" AssemblyFile="..\CommunityToolkit.Maui.BuildTasks\bin\$(Configuration)\$(NetVersion)\CommunityToolkit.Maui.BuildTasks.dll" />
+
+  <Target Name="VerifyMauiWorkloadVersionTask" BeforeTargets="Build">
+    <VerifyMauiWorkloadVersionTask MinimumRequiredMauiVersion="$(RequiredMauiWorkloadVersion)" />
```

Comment:
The property defines above is named `_RequiredMauiWorkloadVersion`, with underscore.